crates/proposer-client/src/rpc.rs

```diff
@@ -81,6 +92,7 @@ impl From<AggSpanProofProposerRequest> for ProposerRequest {
 /// Response for the external proposer `request_span_proof` call
 #[derive(Serialize, Deserialize, Debug)]
 pub struct AggSpanProofProposerResponse {
+    #[serde(rename = "proof_request_id")]
```

Comment:
Why is this snake case and `AggSpanProofProposerRequest` members are camel case?

Reply:
The API is made that way